src/cmd/services/m3coordinator/downsample/metrics_appender.go

```diff
@@ -42,6 +46,17 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+var (
+	m3TypeTag      = []byte("__m3_type__")
+	m3CounterValue = []byte("counter")
+	m3GaugeValue   = []byte("gauge")
+	m3TimerValue   = []byte("timer")
+
+	m3MetricsPrefix              = []byte("__m3")
+	m3MetricsPrefixString        = string(m3MetricsPrefix)
+	m3MetricsGraphiteAggregation = []byte("__m3_graphite_aggregation__")
```

Comment:
nit: Can you make this use a shared M3 prefix variable to avoid them diverging in future? (can accept my suggestion if you think that's ok)
```suggestion
	m3MetricsPrefix       = []byte("__m3")
	m3MetricsPrefixString = string(m3MetricsPrefix)

	m3TypeTag      = []byte(m3MetricsPrefixString + "_type__")
	m3CounterValue = []byte("counter")
	m3GaugeValue   = []byte("gauge")
	m3TimerValue   = []byte("timer")

	m3MetricsGraphiteAggregation = []byte(m3MetricsPrefixString + "_graphite_aggregation__")
```